Sources/NIOSSL/TLSConfiguration.swift

```diff
@@ -268,6 +268,7 @@ public struct TLSConfiguration {
 
     /// Whether to verify remote certificates.
     public var certificateVerification: CertificateVerification
+    public var certificateRequired: Bool = true
```

Comment:
Can you add a doc comment to this explaining what it does and how it interacts with the above?